python/mcap-ros2-support/mcap_ros2/_dynamic.py

```diff
@@ -501,12 +516,14 @@ def _for_each_msgdef(
 ) -> None:
     cur_schema_name = schema_name
 
+    # Remove empty lines
+    schema_text = os.linesep.join([s for s in schema_text.splitlines() if s.strip()])
```

Comment:
what's the purpose of this change?

Reply:
This avoids the 
```py
        if not cur_schema_text:	
            continue
```
which was removed below. Even if our schema is empty, we still want to call `parse_message_string(...)`. 

What I could alternatively, is to add an if before the loop:
```py
if not schema_text.strip():
   # Empty message definition
   ...
   msgdef = parse_message_string(pkg_name, msg_name, cur_schema_text)
   fn(cur_schema_name, short_name, msgdef)
   return
```